notebooks/official/matching_engine/sdk_matching_engine_for_indexing.ipynb

```diff
@@ -98,171 +98,172 @@
     {
```

Comment:
<p><code>Change to </code></p><p> </p><p>"To learn more about configuring the index, see <a href="https://cloud.google.com/vertex-ai/docs/matching-engine/match-eng-setup#input-data-format" target="_blank">Input data format and structure</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1249/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>